scio-core/src/main/scala/com/spotify/scio/coders/instances/kryo/GrpcSerializer.scala

```diff
@@ -60,3 +63,51 @@ private[coders] class StatusRuntimeExceptionSerializer extends KSerializer[Statu
     new StatusRuntimeException(status, trailers)
   }
 }
+
+private[coders] class GaxApiExceptionSerializer extends KSerializer[ApiException] {
+  private lazy val statusSer = new StatusSerializer()
+  override def write(kryo: Kryo, output: Output, e: ApiException): Unit = {
+    kryo.writeObject(output, e.getMessage)
+    kryo.writeClassAndObject(output, e.getCause)
+    e.getStatusCode match {
+      case grpc: GrpcStatusCode =>
+        kryo.writeClass(output, classOf[GrpcStatusCode])
+        kryo.writeObject(output, grpc.getTransportCode.toStatus, statusSer)
+      case http: HttpJsonStatusCode =>
+        kryo.writeClass(output, classOf[HttpJsonStatusCode])
+        kryo.writeObject(output, http.getTransportCode)
```

Comment:
not really part of gRPC but since `gax-httpjson` is in the classpath, handling this type too